ios/sdk/src/JitsiMeetView.m

```diff
@@ -87,157 +66,116 @@ - (instancetype)initWithFrame:(CGRect)frame {
     [externalAPI sendSetClosedCaptionsEnabled:enabled];
 }
 
-#pragma mark Private methods
-
-/**
- * Passes the given props to the React Native application. The props which we pass
- * are a combination of 3 different sources:
- *
- * - JitsiMeet.defaultConferenceOptions
- * - This function's parameters
- * - Some extras which are added by this function
- */
-- (void)setProps:(NSDictionary *_Nonnull)newProps {
-    NSMutableDictionary *props = mergeProps([[JitsiMeet sharedInstance] getDefaultProps], newProps);
-
-    // Set the PiP flag if it wasn't manually set.
-    NSMutableDictionary *featureFlags = props[@"flags"];
-    if (featureFlags[PiPEnabledFeatureFlag] == nil) {
-        featureFlags[PiPEnabledFeatureFlag]
-            = [NSNumber numberWithBool:
-               self.delegate && [self.delegate respondsToSelector:@selector(enterPictureInPicture:)]];
-    }
+#pragma mark Private
 
-    props[@"externalAPIScope"] = externalAPIScope;
-
-    // This method is supposed to be imperative i.e. a second
-    // invocation with one and the same URL is expected to join the respective
-    // conference again if the first invocation was followed by leaving the
-    // conference. However, React and, respectively,
-    // appProperties/initialProperties are declarative expressions i.e. one and
-    // the same URL will not trigger an automatic re-render in the JavaScript
-    // source code. The workaround implemented below introduces imperativeness
-    // in React Component props by defining a unique value per invocation.
-    props[@"timestamp"] = @(mach_absolute_time());
-
-    if (rootView) {
-        // Update props with the new URL.
-        rootView.appProperties = props;
-    } else {
-        RCTBridge *bridge = [[JitsiMeet sharedInstance] getReactBridge];
-        rootView
-            = [[RNRootView alloc] initWithBridge:bridge
-                                      moduleName:@"App"
-                               initialProperties:props];
-        rootView.backgroundColor = self.backgroundColor;
-
-        // Add rootView as a subview which completely covers this one.
-        [rootView setFrame:[self bounds]];
-        rootView.autoresizingMask
-            = UIViewAutoresizingFlexibleWidth
-                | UIViewAutoresizingFlexibleHeight;
-        [self addSubview:rootView];
-    }
+- (BOOL)isPiPEnabled {
+    return self.delegate && [self.delegate respondsToSelector:@selector(enterPictureInPicture:)];
 }
 
-+ (BOOL)setPropsInViews:(NSDictionary *_Nonnull)newProps {
-    BOOL handled = NO;
+- (void)registerObservers {
+    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(handleSendEventNotification:) name:@"JitsiMeet.SendEvent" object:nil];
+}
 
-    if (views) {
-        for (NSString *externalAPIScope in views) {
-            JitsiMeetView *view
-                = [self viewForExternalAPIScope:externalAPIScope];
+- (void)handleSendEventNotification:(NSNotification *)notification {
```

Comment:
Here we should filter notifications by external API scope. I know we don't really support multiple views with meetings at the same time, but the idea is that one might show a new view while the other one is being destroyed or something.

Alternatively we can completely get rid of it...

Reply:
removed the multiple views support